testSrc/unit/io/flutter/run/SdkFieldsTest.java

```diff
@@ -21,18 +21,22 @@
   public void shouldReadFieldsFromXml() {
     final Element elt = new Element("test");
     addOption(elt, "filePath", "lib/main.dart");
+    addOption(elt, "programArgs", "--debug");
```

Comment:
Do we need to make sure the user doesn't set flags that are important for debugging? (For example, passing in --no-hot will interfere with hot reloading.)